astro_2.0/features/CreateProposal/components/ErrorIndicator/ErrorIndicator.module.scss

```diff
@@ -0,0 +1,15 @@
+.root {
+  left: 12px;
+  position: absolute;
+  top: -2px;
+}
+.icon {
+  color: var(--color-error);
+  margin-left: 8px;
+  transform: translateY(-1px);
+}
+
+.error.error {
```

Comment:
I consider it's an accident duplication.

Reply:
No, this just a way to increase specificity.  Otherwise my error class still overriden by component styles